easel/lib/screens/creator_hub/creator_hub_screen.dart

```diff
@@ -221,36 +230,32 @@ class _CreatorHubContentState extends State<CreatorHubContent> {
                     onGridSelected: (context) => BuildNFTsContent(
                         onDraftList: (context) => BuildGridView(
                               nftsList: viewModel.nftDraftList,
-                              onEmptyList: (context) => Padding(
-                                padding: EdgeInsets.symmetric(horizontal: 20.w),
-                                child: getEmptyDraftListWidget(),
-                              ),
+                              onEmptyList: (context) => getDraftPublishListEmptyWidget(isForDraft: true),
                             ),
                         onPublishedList: (context) => BuildGridView(
                               nftsList: viewModel.nftPublishedList,
-                              onEmptyList: (context) => Padding(
-                                padding: EdgeInsets.symmetric(horizontal: 20.w),
-                                child: getEmptyPublishedWidget(),
-                              ),
+                              onEmptyList: (context) => getDraftPublishListEmptyWidget(isForDraft: false),
                             ),
                         collectionType: viewModel.selectedCollectionType),
                     onListSelected: (context) => BuildNFTsContent(
-                        onDraftList: (context) => Padding(
-                              padding: EdgeInsets.symmetric(horizontal: 20.w),
-                              child: BuildListView(
-                                nftsList: viewModel.nftDraftList,
-                                onEmptyList: (context) => getEmptyDraftListWidget(),
-                              ),
+                        onDraftList: (context) => BuildListView(
+                              nftsList: viewModel.nftDraftList,
+                              onEmptyList: (context) => getDraftPublishListEmptyWidget(isForDraft: true),
                             ),
                         onPublishedList: (context) => BuildListView(
                               key: ValueKey(viewModel.nftPublishedList.length),
                               nftsList: viewModel.nftPublishedList,
-                              onEmptyList: (context) => getEmptyPublishedWidget(),
+                              onEmptyList: (context) => getDraftPublishListEmptyWidget(isForDraft: false),
                             ),
                         collectionType: viewModel.selectedCollectionType),
                     viewType: viewModel.viewType,
                   ),
                 ),
+                if (viewModel.selectedCollectionType == CollectionType.draft || (viewModel.selectedCollectionType == CollectionType.published && viewModel.nftPublishedList.isEmpty))
```

Comment:
If there are drafts do we still show the create draft button?

Reply:
yes

<img width="930" alt="image" src="https://user-images.githubusercontent.com/115459890/204502848-6fff69cd-42bd-42a1-b39a-60d853b180b3.png">